atc/db/dbfakes/fake_check_lifecycle.go

```diff
@@ -2,29 +2,31 @@
 package dbfakes
 
 import (
-	"sync"
-	"time"
+	sync "sync"
```

Comment:
was that a formatting issue? I'm not sure we need to rename these imports to the same package names 🤔

Reply:
It is a fake file that is generated we got no control here.

Reply:
whoops that's true 😅